cognite/neat/_session/_prepare.py

```diff
@@ -287,6 +288,33 @@ def convert_data_type(self, source: tuple[str, str], *, convert: Callable[[Any],
         transformer = ConvertLiteral(subject_type, subject_predicate, convert)
         self._state.instances.store.transform(transformer)
 
+    def property_to_type(self, source: tuple[str | None, str], type: str, new_property: str | None = None) -> None:
+        """Convert a property to a new type.
+
+        Args:
+            source: The source of the conversion. A tuple of (type, property)
+                    where property is the property that should be converted.
+                    You can pass (None, property) to covert all properties with the given name.
+            type: The new type of the property.
+            new_property: Add the identifier as a new property. If None, the new entity will not have a property.
+
+        Examples:
```

Comment:
so that mkdocs picks up the example for formatting
```suggestion
        Example:
```